<launch>
	<!-- PARAMETERS FOR SETTING UP ROBOT STATE-->
	<!-- Parameter for the battery controller execution mode -->
	<param name="battery_controller_execution_mode" type="str" value="MANUAL" />
    <!-- Parameter for the initial battery level -->
    <param name="battery_intial_value" type="int" value="100" />

	<!-- LAUNCHING NODE -->
    <node name="robot_state" pkg="exprob_assignment_1" type="robot_state.py" output="screen"/>
</launch>
